crypto/src/lib.rs

```diff
@@ -223,33 +256,65 @@ impl MlsCentral {
     }
 
     /// Sets the consumer callbacks (i.e authorization callbacks for CoreCrypto to perform authorization calls when needed)
+    ///
+    /// # Arguments
+    /// * `callbacks` - a callback to be called to perform authorization
+    ///
+    /// # Errors
+    /// This function currently can't cause any errors
     pub fn callbacks(&mut self, callbacks: Box<dyn CoreCryptoCallbacks>) -> CryptoResult<()> {
         self.callbacks = Some(callbacks);
         Ok(())
     }
 
     /// Returns the client's public key as a buffer
+    /// # Errors
+    /// This function currently can't cause any errors
```

Comment:
Then you should modify the return type

Reply:
> Then you should modify the return type

We can do some housekeeping later on on those. Those methods return `Result`s because in the past all of those properties were behind `RwLock`s and `Mutex`es, so the operation *could* potentially fail in the case of a poisoned lock.